clang/test/SemaCXX/cxx20-warn-dangling-paren-list-agg-init.cpp

```diff
@@ -0,0 +1,42 @@
+// RUN: %clang_cc1 -verify -std=c++20 %s -fsyntax-only
+
+namespace std {
+template <class T> struct remove_reference { typedef T type; };
+template <class T> struct remove_reference<T&> { typedef T type; };
+template <class T> struct remove_reference<T&&> { typedef T type; };
+
+template <class T> typename remove_reference<T>::type &&move(T &&t);
+} // namespace std
+
+// dcl.init 16.6.2.2
+struct A {
```

Comment:
Can you also add more nesting:
https://godbolt.org/z/9jbsxPjrW
https://godbolt.org/z/cj97eaPsK

Reply:
Thanks for the great examples. Added.